packages/create-plugin/templates/common/.config/README.md

```diff
@@ -139,3 +139,26 @@ We need to update the `scripts` in the `package.json` to use the extended Webpac
 -"dev": "webpack -w -c ./.config/webpack/webpack.config.ts --env development",
 +"dev": "webpack -w -c ./webpack.config.ts --env development",
 ```
+
+### Configure grafana image to use when running docker
+
+By default `grafana-enterprise` will be used as the docker image for all docker related commands. If you want to override this behaviour simply alter the `docker-compose.yaml` by adding the following build arg `grafana_image`. 
+
+**Example:**
+
+```yaml
+version: '3.7'
+
+services:
+  grafana:
+    container_name: 'myorg-basic-app'
+    build:
+      context: ./.config
+      args:
+        grafana_version: ${GRAFANA_VERSION:-9.1.2}
+        grafana_image: ${GRAFANA_IMAGE:-grafana}
```

Comment:
I'm wondering if we should set the default `grafana_image` in the [docker-compose.yaml](https://github.com/grafana/plugin-tools/blob/main/packages/create-plugin/templates/common/docker-compose.yaml#L8-L9) to `grafana` and then update this doc to mention switching to `grafana-enterprise` with the diff for using `grafana-enterprise`?

e.g.

```
args:
  grafana_version: ${GRAFANA_VERSION:-9.1.2}
-  grafana_image: ${GRAFANA_IMAGE:-grafana}
+  grafana_image: ${GRAFANA_IMAGE:-grafana-enterprise}
```
Originally I thought switching to enterprise would be an easy solution to this but now I'm thinking it's more internal to Grafana and might confuse the community?

Reply:
Will leave it as it is according to our chat, ok?